lib/web_ui/lib/src/engine/clipboard.dart

```diff
@@ -91,9 +91,49 @@ class ClipboardAPIPasteStrategy implements PasteFromClipboardStrategy {
 class ExecCommandCopyStrategy implements CopyToClipboardStrategy {
   @override
   void setData(String text) {
-    // TODO(nurhan): https://github.com/flutter/flutter/issues/48578
-    print('Clipboard is only implemented for browsers '
-        'supporting Clipboard API. Use context menu for text editing.');
+    final html.TextAreaElement tempTextArea = _appendTemporaryTextArea();
```

Comment:
It would be helpful to add a comment explaining the approach and maybe a link to MDN or something.

Reply:
Thanks! I'll add this one: https://developers.google.com/web/updates/2015/04/cut-and-copy-commands

Reply:
done.